inc/commondbtm.class.php

```diff
@@ -1391,19 +1391,78 @@ function prepareInputForAdd($input) {
       return $input;
    }
 
-    /**
+   /**
+    * Compute the name of a copied item
+    *
+    * @param string   $current_item The item being copied
+    * @param null|int $copy_index   The index to append to the name
+    *
+    * @return string The computed name of the new item to be created
+    */
+   public function computeCloneName(
+      string $current_name,
+      ?int $copy_index = null
+   ): string {
+      $base_name = sprintf(__("Copy of %s"), $current_name);
+
+      if (is_null($copy_index)) {
+         return $base_name;
+      }
+
+      // Note, we can't use sprintf(__("Copy of %s (%d)"), $current_name, $copy_index) here
+      // We need to rely on the same translation to garantee uniqueness
+      return $base_name . " ($copy_index)";
```

Comment:
IMHO, using a prefix is not the best option. Indeed, in many cases, items are ordered by name, so, if prefix is used, an item an its clone may be distant from each other in a list.
If a suffix is used instead of a prefix (e.g. `My item (copy)`), an item will be followed by its clones in an alphabetically ordered list.

Reply:
Indeed, I agree with this.  
I only went for the "Copy My item" approach because it was already used for some items (rules).  

If we are allowed to change this format then how about one of the following:
1) My item (1)
2) My item - copy (1)
3) My item (copy 1)

I like 1 or 2, not sure about 3.

Reply:
Solution 1 is the one used by Chrome (at least on my Ubuntu system) when downloading a file multiple times.

Solution 3 is the one used by the default Ubuntu file browser:
![image](https://user-images.githubusercontent.com/33253653/154924582-1f8b3ebe-d2f8-4a88-a6f4-2c6243e523b3.png)
If I remeber right, it is also the system used by Windows file browser.

Problem with solution 1 is that it may be ambiguous with the format used to append ID on items. If you see `Some computer (3)` in GLPI UI, will it referes to the `Some computer` which ID is `3`, or will it refers to the third clone of this item ?

Reply:
As an additional info, Windows files explorer use the first solution but start the index at 2:
![image](https://user-images.githubusercontent.com/42734840/154927004-fe9e3a48-acab-48ad-b9ca-7563e4d2e4fa.png)

Understood for the confusion with the ID part, it's indeed not ideal.

If we follow Ubuntu implementation of the third solution we end up with:
- My item
- My item (copy)
- My item (2nd copy) 
- My item (3rd copy) 
- My item (4th copy) 

Would this be OK ?
No issues with translation for the dynamic 2nd/3rd/4th part ?

Reply:
I guess `(copy)`, `(copy 2)`, `(copy 3)`, ... would be easier to handle as it would not require to handle translation of each number.

Reply:
Ok for me.
Is this good for you too @orthagh @trasher ?